src/import-service/handler.ts

```diff
@@ -0,0 +1,9 @@
+import {
+  importProductsFile,
+  importFileParser
+} from './src/handlers';
+
+export {
+  importProductsFile,
+  importFileParser
+}
```

Comment:
`export {
  importProductsFile,
  importFileParser
} from './src/handlers';`
It will also work.